### PR TITLE
update v2 to use newest onflow packages

### PR DESCRIPTION
**Pull request by bthaile**
Closes: #???

## Description

Daft branch to upgrade to use cadence v1 preview packages.

______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md).
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```
Comment (janezpodhostnik) [inline .vscode/launch.json]: did you intend to add this file?

File: model/event.go [hunk 2]
```diff
@@ -39,14 +38,9 @@ func EventsFromFlow(flowEvents []flow.Event) ([]Event, error) {
 }
 
 func parseEvent(event flow.Event) (Event, error) {
-	values := make([]string, len(event.Value.Fields))
-	for j, field := range event.Value.Fields {
-		encoded, _ := jsoncdc.Encode(field)
-		values[j] = string(encoded)
-	}
-
+	// do events have more than one value, this seemed to have changed from previous version
```
Comment (janezpodhostnik) [inline model/event.go]: No they shouldn't.  events hold only one cadence event value, but that can have multiple fields.

Comment (bthaile) [reply model/event.go]: thanks, stumbling my way through upgrading libraries.

File: .vscode/launch.json (hunk 1 above)
Comment (bthaile) [reply .vscode/launch.json]: nope will update .gitignore

**Review by janezpodhostnik — APPROVED** [review]

**Pull request closed without merging**